awscli/botocore/auth.py

```diff
@@ -880,7 +881,7 @@ def add_auth(self, request):
 # the botocore.crt.auth module imports functions/classes defined above from
 # this module. In the future, we should isolate those functions/classes into
 # a separate utility module to avoid any potential circular import.
-import botocore.crt.auth
+import botocore.crt.auth  # noqa
```

Comment:
The AWS CLI v2 has a requirement on `awscrt`, so this is implemented differently than `botocore`. Left this for a fix for another time.